### PR TITLE
Forms Automated Test - LPS-131372 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase [hunk 1]
```diff
@@ -197,14 +197,65 @@ definition {
...
+			locator1 = "FormFields#SELECT_FIELD");
+
+		var iterator = "1";
+
+		for (var option : list "A,B,C") {
+			Form.viewSelectFromListOnPreview(
+				fieldName = "Option ${option}",
+				rowPosition = "${iterator}");
+
+			var iterator = MathUtil.sum("${iterator}", "1");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase:239]: @ewertonluna, does the reserved word `var` is really needed here? Won't it redeclare the variable `iterator` already declared on line **246**?

If so, can you please fix it?

File: portal-web/test/functional/com/liferay/portalweb/macros/Form.macro [hunk 2]
```diff
@@ -721,10 +721,17 @@ definition {
 			value1 = "Draft");
 	}
 
+	macro viewSelectFromListOnPreview {
+		AssertElementPresent(
+			key_fieldName = "${fieldName}",
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/Form.macro:726]: @ewertonluna, let's rename to keep a naming consistency:

- `viewSelectFromListOnPreview` to `viewSelectFromListOptionOnPreview`
- `key_fieldName` to `key_optionValue`
- `fieldName` to `optionValue`

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@ewertonluna, can you please rebase your code against the latest branch master to solve the reported conflicts?

Thanks.

**Comment by natocesarrego** [comment]
OMG! Rebase conflicts again. :disappointed: 

@ewertonluna, can you please rebase against Brian's latest master?

Once you do it, feel free to trigger `ci:forward`, after all your pull looks good to go. :wink:

**Comment by natocesarrego** [comment]
If your have any doubt, please let me know.

**Comment by pedro-michel** [comment]
ci:forward

**Pull request closed without merging**